nondex-gradle-plugin/src/main/groovy/edu/illinois/nondex/gradle/plugin/NonDexJavaPlugin.groovy

```diff
@@ -0,0 +1,43 @@
+package edu.illinois.nondex.gradle.plugin
+
+import edu.illinois.nondex.gradle.plugin.tasks.GenerateOutputJar
+import edu.illinois.nondex.gradle.plugin.tasks.NonDexPrepare
+import edu.illinois.nondex.gradle.plugin.tasks.NonDexTest
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+
+
+class NonDexJavaPlugin implements Plugin<Project> {
+    public static final String NONDEX_JAVA_CONFIGURATION_NAME = 'nondexJava'
+    public static final String NONDEX_JAVA_VERSION = '1.1.1'
```

Comment:
Use the version by reading the static field in ConfigurationDefaults... did you undo my changes on your local computer? We did some of these already.